docs/x509/verification.rst

```diff
@@ -30,6 +30,33 @@ chain building, etc.
     :class:`cryptography.x509.general_name.DNSName`,
     :class:`cryptography.x509.general_name.IPAddress`.
 
+.. class:: ServerVerifier
+
+    .. versionadded:: 42.0.0
+
+    A ServerVerifier verifies server certificates.
+
+    It contains and describes various pieces of configurable path
+    validation logic, such as which subject to expect, how deep prospective
+    validation chains may go, which signature algorithms are allowed, and
+    so forth.
+
+    ServerVerifier instances cannot be constructed directly;
+    :class:`PolicyBuilder` must be used.
+
+    .. attribute:: subject
+
+        :type: :class:`Subject`
+
+        The verifier's subject.
+
+    .. attribute:: validation_time
+
+        :type: :class:`datetime.datetime`
+
+        The verifier's validation time. If not specified during construction
+        (see :class:`PolicyBuilder`), this is set to
+        :meth:`datetime.datetime.now`.
```

Comment:
The `if not specified` should go in the policy builder docs, and should state that it's now at the time you call build

Reply:
[2e7bd70](https://github.com/pyca/cryptography/pull/9405/commits/2e7bd70c01465a4983e1e9a3e8ce0d4a930055d2)